.github/workflows/publish-master-npm.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish NPM Package `master`
+on:
+  push:
```

Comment:
I think we don't want to publish a new npm package from every push. This should be "manually triggered" or something like that.

Reply:
Why not? Because master might be broken? If that is the case you can always choose a specific hash

Reply:
It would flood npmjs with versions that have exactly the same contents (not a good practice). Not sure if we can detect here PR labels (on merge). If yes -> we could publish when PR has `area/extensions` label.